src/erlcloud_cloudwatch_logs.erl

```diff
@@ -465,7 +465,7 @@ log_stream_order_by(last_event_time) -> <<"LastEventTime">>.
     log_stream_name(),
     seq_token(),
     events()
-) -> datum:either( seq_token() ).
```

Comment:
I don't know where this comes from, but I "fixed" it to what I could analyze by looking at the code and tests.

Reply:
+1 to what you have, I have no idea what this means.